sklearn/linear_model/_coordinate_descent.py

```diff
@@ -754,22 +754,28 @@ def enet_path(
 
 
 class ElasticNet(MultiOutputMixin, RegressorMixin, LinearModel):
-    """Linear regression with combined L1 and L2 priors as regularizer.
+    r"""Linear regression with combined L1 and L2 priors as regularizer.
```

Comment:
I don't think that a raw string is needed. If you take a look at [NMF](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.NMF.html), which is properly rendered and uses latex syntax, it's a usual string.
```suggestion
    """Linear regression with combined L1 and L2 priors as regularizer.
```